src/en/join-us/index.md

```diff
@@ -7,7 +7,12 @@ The contribution is {{ general-info.contribution }} (including 21% VAT). Members
 
 The contribution for students at a higher educational institution is {{ general-info.students_contribution }} (including VAT). Students who register after June 30 pay {{ general-info.students_discounted }}. 
 
-Naturally, we will treat your data confidentially. Only the board and the membership administration (appointed by the board) will see this data. Your name and possible employer will be published on the public membership list, if desired.
+<div style="padding: var(--spacing-half) var(--spacing); background: #efefef;">
```

Comment:
Is this newly introduced `div` (with a `<br />` after the closing tag) really necessary? Markdown should create its own spacing, right? (same remark for the Dutch version)